sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -282,6 +282,16 @@ case class InsertIntoStatement(
 case class ShowTablesStatement(namespace: Option[Seq[String]], pattern: Option[String])
   extends ParsedStatement
 
+/**
+ * A CREATE NAMESPACE statement, as parsed from SQL.
+ */
+case class CreateNamespaceStatement(
+  namespace: Seq[String],
+  ifNotExists: Boolean,
+  comment: Option[String],
+  locationSpec: Option[String],
+  properties: Map[String, String]) extends ParsedStatement
```

Comment:
@cloud-fan, what is the behavior of IF NOT EXISTS with DBPROPERTIES? If the namespace exists, are properties updated?

Reply:
same as table/database, if table/database doesn't exist, the command is a noop.